aw_client/client.py

```diff
@@ -243,7 +243,7 @@ class RequestQueue(threading.Thread):
 
     VERSION = 1  # update this whenever the queue-file format changes
 
-    def __init__(self, client, dispatch_interval=0):
+    def __init__(self, client: ActivityWatchClient, dispatch_interval: float=0) -> None:
```

Comment:
Remove default dispatch_interval from class parameter? If it's only for testing aw-client you could just as well just hardcode it.

Reply:
`dispatch_interval` was never used in the class, removed.